srcpkgs/python3-transifex-client/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-transifex-client'
+pkgname=python3-transifex-client
+version=0.13.6
+revision=1
+wrksrc="transifex-client-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-urllib3 python3-requests python3-six python3-slugify"
```

Comment:
use version specifiers for slugify to help dissuade people from updating it.

Reply:
Fixed.